guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,48 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+You can select to resolve package dependencies per Content View or have it enabled by default via Settings -> Content -> "content view Dependency Solving Default".
+{Team} does not recommend enabling content view dependency solving by default unless the pitfalls are well-understood.
 
-Note that resolving package dependencies can cause significant delays to Content View promotion.
-The package dependency resolution feature does not consider packages that are installed on your system independently of the Content View or solve dependencies across repositories.
+Note that resolving package dependencies can cause significant delays to Content View publication.
+The more content and repositories in the content view, the greater the delay.
+
+The package dependency resolution feature does not take which packages are installed on your systems into account.
+Only packages within the content view's repositories are considered.
+For example, if a content view has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
```

Comment:
```suggestion
ifeval::["{client-os-family}" == "Red Hat"]
For example, if a content view has only AppStream in it, dependency solving will not know to pull dependent BaseOS content in at publish time.
endif::[]
```

This example depends on both the content type (Yum) and OS of the managed host (EL). See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L134

Its set for all build targets but allows ATIX in downstream to overwrite it for OS-specific guides; i.e. SLES and Debian/Ubuntu would not should this line in downstream.

Reply:
I have some very specific RHEL references later on, should those also be locked into the client os family of "Red Hat" ?

Reply:
Yes, makes sense to me.